### ISSUE TITLE
macos: Failed startup for Docker 17.12.0-ce-mac46 (21698)

### ISSUE THREAD

**Issue by thisisthekap** [Microsoft/mssql-docker#235]
When starting the MSSQL container using the latest Docker version (17.12.0-ce-mac46 (21698)), the startup fails with the following message (using the `latest` tag):

```
ERROR: for 37ee84e1c567_devsqlserver_dev-sql-server_1  Cannot start service dev-sql-server: driver failed programming external connectivity on endpoint devsqlserver_dev-sql-server_1 (50efb94107253747e340acf9ca7f4bb51e3d38f8916fc6022bcc1317d2026254): Error starting userland proxy: mkdir /port/tcp:0.0.0.0:1433:tcp:172.18.0.2:1433: file exists
```

When I remove the port mapping from my `docker-compose.yml` file, the container starts up successfully. Other containers (with port mapping) start up flawlessly.

**Comment by thisisthekap** [comment]
As an intermediate workaround: I just found out on https://github.com/docker/for-mac/issues/1120 that old Docker releases can easily be downloaded from https://docs.docker.com/docker-for-mac/release-notes/

**Comment by thisisthekap** [comment]
The issue seems to be solved with Docker 17.12.0-ce-mac47 (21805). I am closing it.
